RetailCoder.VBE/Inspections/VariableRequiresSetAssignmentEvaluator.cs

```diff
@@ -0,0 +1,160 @@
+﻿using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Inspections
+{
+    class VariableRequiresSetAssignmentEvaluator
```

Comment:
IMHO about 75% of the `DeclarationFinder` is "unexpected" capability, I don't really have a strong opinion one way or another.  Some of the other functionality (bool tests on multiple properties) could be moved to `Declaration`, but that's getting huge too.  Separate static class is going to be a lot easier to manage in the short term though.